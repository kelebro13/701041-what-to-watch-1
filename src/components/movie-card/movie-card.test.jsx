import React from "react";
import renderer from "react-test-renderer";
import MovieCard from "./movie-card.jsx";

it(`render correctly MovieCard component`, () => {
  const title = `Aviator`;
  const clickHandler = jest.fn();

  const tree = renderer
    .create(<MovieCard title={title} onClick={clickHandler}/>)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
