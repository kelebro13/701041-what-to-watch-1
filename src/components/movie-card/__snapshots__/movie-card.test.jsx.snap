// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render correctly MovieCard component 1`] = `
<article
  className="small-movie-card catalog__movies-card"
  onMouseEnter={[Function]}
  onMouseLeave={[Function]}
>
  <a
    href="/film/1"
    onClick={[Function]}
  >
    <div
      className="small-movie-card__image"
    />
  </a>
  <h3
    className="small-movie-card__title"
  >
    <a
      className="small-movie-card__link"
      href="/film/1"
      onClick={[Function]}
    >
      The Grand Budapest Hotel
    </a>
  </h3>
</article>
`;
